test/src/test/java/hudson/logging/LogRecordManagerRealTest.java

```diff
@@ -0,0 +1,34 @@
+package hudson.logging;
+
+import java.util.List;
+import java.util.logging.LogRecord;
+import jenkins.model.Jenkins;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.RealJenkinsRule;
+
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Checkstyle complaining

Reply:
aware, just left it while you were debugging